dockstore-webservice/src/main/java/io/dockstore/webservice/api/VerifyRequest.java

```diff
@@ -1,48 +0,0 @@
-/*
```

Comment:
It's odd how in the GitHub UI this file shows a diff with everything removed, but DockerHubRegistry.java shows as deleted. But the logs seem right, I guess a GitHub bug.